product_docs/docs/pgd/5/durability/legacy-sync.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Legacy synchronous replication using PGD
+navTitle: Legacy synchronous replication
+---
+
+!!! Warning Important
+Consider using [Group Commit](group-commit) or [Synchronous Commit](synchronous_commit) instead.
```

Comment:
I would reword this to highly recommend commit scope based synchronous commit, it's same thing with more PGD-aware config, the group commit, whatever, that does not need to be pushed here much.